jcl/src/java.base/share/classes/java/lang/StackTraceElement.java

```diff
@@ -96,6 +97,14 @@ public StackTraceElement(String classLoaderName, String module, String version,
 public String getClassLoaderName() {
 	return classLoaderName;
 }
+
+/**
+ * Returns whether the ClassLoader used to load the class for the method in the stack frame is a built-in ClassLoader.
+ * @return true if the ClassLoader provided is a built-in ClassLoader, false otherwise.
+ */
+public boolean getClassLoaderIsBuiltIn() {
```

Comment:
Public methods outside of the java spec can't be added.

Reply:
Ah oops right, hence you mentioning the result should get passed in to `printStackTraceElement()`. Fixing.